packages/runtime-core/src/components/BaseTransition.ts

```diff
@@ -199,7 +199,7 @@ const BaseTransitionImpl: ComponentOptions = {
       }
 
       const oldChild = instance.subTree
-      const oldInnerChild = oldChild && getInnerChild(oldChild)
+      let oldInnerChild = oldChild && getInnerChild(oldChild)
```

Comment:
GitHub doesn't let me remove unmodified lines, but removing the previous ``oldChild`` and replacing it with this line:
```suggestion
      let oldInnerChild = instance.subTree && getInnerChild(instance.subTree)
```

Disposes the old node immediately.

Reply:
done